api/config.py

```diff
@@ -84,22 +85,22 @@
 ).get_default_database()
 log.debug(str(db))
 
+es = elasticsearch.Elasticsearch([__config['persistent']['elasticsearch_host']])
```

Comment:
Yes. A more flexible approach would be to determine the es host in `SearchHandler.get()`. Seems like the right thing to do. Any objections?